config/webpack.config.js

```diff
@@ -141,7 +141,8 @@ module.exports = function (env, isomorphic = false, noAnimation = false, ilibAdd
 										remove: false
 									},
 									stage: 3,
-									features: {'custom-properties': false}
+									features: {'custom-properties': false},
+									enableClientSidePolyfills: true
```

Comment:
Could you explain more in detail why we need this option?

Reply:
This change is from [this link](https://github.com/csstools/postcss-plugins/wiki/PostCSS-Preset-Env-8#postcss-preset-env-breaking-changes).

This disables by default the following plugins:

[PostCSS Blank Pseudo](https://github.com/csstools/postcss-plugins/tree/main/plugins/css-blank-pseudo)
[PostCSS Focus Visible](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-focus-visible)
[PostCSS Focus Within](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-focus-within)
[PostCSS Has Pseudo](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-has-pseudo)
[Prefers Color Scheme Query](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-prefers-color-scheme)

I added this option to ensure the same work as the previous `postcss-preset-env`, but I will review if it is okay to remove it as it is not necessary.

Reply:
The current version of chromium we are using seems okay not to turn this on.